frontend/src/metabase/components/ArchivedItem/ArchivedItem.tsx

```diff
@@ -65,7 +70,7 @@ export const ArchivedItem = ({
           </Tooltip>
         )}
         {onDelete && (
-          <Tooltip tooltip={t`Delete this ${type}`}>
+          <Tooltip tooltip={t`Delete this ${getTranslatedEntityName(model)}`}>
```

Comment:
@alxnddr thanks for the tip about the `getTranslatedEntityName` helper. I didn't even know we have it.

Just a final nit - do you mind if I lowercase the output?
It currently looks like this:
![image](https://github.com/metabase/metabase/assets/31325167/16d6a421-86ca-4a96-b949-fcddfc1dc0d4)

Reply:
Yep, you can lowercase the translated string